dotCMS/src/main/java/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -2270,6 +2270,8 @@ public List<Map<String, Object>> getFolderSubfolders(final String parentFolderId
 		final Role[] roles = DwrUtil.getUserRoles(user);
 		final Folder parentFolder = this.folderAPI.find(parentFolderId, user, false);
 		final List<Folder> subFolders = this.folderAPI.findSubFolders(parentFolder, user, false);
+		// Sort the list by folder name
+		subFolders.sort(Comparator.comparing(Folder::getName));
```

Comment:
Need to also make sure this work is in the BrowserResource or we will have a regression as soon as we rewrite the browser ui.

Reply:
@AndreyDotcms let's try to do a postman test... if you have any questions don't hesitate to reach me.

Reply:
Donde https://github.com/dotCMS/core/pull/26281/commits/352e76585600ab84437be63f5fe246077943cf9d